readthedocs/templates/projects/project_version_detail.html

```diff
@@ -15,6 +16,20 @@
 
 {% block content %}
     <h2> Editing {{ version.slug }} </h2>
+
+    {% if request.user|is_admin:project %}
+      {% if not version.active and version.built %}
+        <p class="empty">
+          {% url "project_version_delete_html" project.slug version.slug as version_delete_url %}
+          {% blocktrans %}
```

Comment:
Use `trimmed` on the blocktrans call to improve the text whitespace handling in gettext